src/markitdown/_markitdown.py

```diff
@@ -712,6 +713,49 @@ def convert(self, local_path, **kwargs) -> Union[None, DocumentConverterResult]:
         return result
 
 
+class DocConverter(HtmlConverter):
+    """
+    Converts DOC files to Markdown.
+    """
+
+    def convert(self, local_path, **kwargs) -> Union[None, DocumentConverterResult]:
+        # Bail if not a DOC
+        extension = kwargs.get("file_extension", "")
+        if extension.lower() != ".doc":
+            return None
+
+        if not (soffice := shutil.which("soffice")):
+            return None
+
+        local_path = Path(local_path)
+        outdir = local_path.parent
+        result = None
+
+        libreoffice_command = [
```

Comment:
Should we be telling users to install these libraries?

Reply:
Or install them for them?

Reply:
```py
        if not (soffice := shutil.which("soffice")):
            return DocumentConverterResult(
                title=None,
                text_content="[ERROR] LibreOffice not found. Please install LibreOffice and try again.",
            )
```

added above